sympy/stats/tests/test_rv.py

```diff
@@ -12,6 +12,7 @@
 from sympy.external import import_module
 from sympy.core.numbers import comp
 from sympy.stats.frv_types import BernoulliDistribution
+from sympy.series.order import O
```

Comment:
What is this for?

Reply:
In one of my previous commits, I was using O(1/n), this term comes in the formula for entropy of Binomial.